OasisDEX/oasis-borrow
# 🧿 Ajna order informations and after pills

# [Ajna order informations and after pills](https://app.shortcut.com/oazo-apps/story/7924/ajna-minor-features)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- added order information based on simulation (not all are defined since library is not ready yet)
- added after pills
  
## How to test 🧪
  <Please explain how to test your changes>

- use ajna feature toggle
- order information in ajna form and after pills should display data from simulation
